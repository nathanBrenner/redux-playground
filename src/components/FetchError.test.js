import React from "react";
import FetchError from "./FetchError";
import { shallow } from "enzyme";
import renderer from "react-test-renderer";

describe("FetchError", () => {
  test("renders without crashing", () => {
    shallow(<FetchError />);
  });

  test("snapshot", () => {
    const component = renderer.create(<FetchError />);
    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
