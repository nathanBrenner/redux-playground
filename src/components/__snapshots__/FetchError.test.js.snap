// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FetchError snapshot 1`] = `
<div>
  <p>
    Could not fetch todos. 
  </p>
  <button
    onClick={undefined}
  >
    Retry
  </button>
</div>
`;
